test/modules/test_nuclei.py

```diff
@@ -73,3 +74,18 @@ def test_links(self) -> None:
             "values in the server response via GET-requests., [medium] http://test-php-xss-but-not-on-homepage:80/xss.php: Fuzzing Parameters - Cross-Site Scripting Cross-site scripting "
             "was discovered via a search for reflected parameter values in the server response via GET-requests.\n",
         )
+
+    def test_group_targets_by_tech(self) -> None:
```

Comment:
one minor comment: I would call it group_targets_by_missing_tech and precisely describe in the docstring that they describe the target they **lack**